apps/web/src/app/dashboard/[name]/settings/layout.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+import type { PropsWithChildren } from "react";
+
+import { cn } from "~/lib/utils";
+
+export default function Settings({ children }: PropsWithChildren) {
+  const pathname = usePathname();
+
+  return (
+    <div className="mx-auto w-full flex-1 px-6 py-8 sm:p-12">
+      <div className=" mx-auto w-full max-w-6xl">
+        <div className="flex h-full flex-col lg:flex-row">
+          <nav className="mr-2 w-full shrink-0 lg:w-1/5">
+            <div className="mt-1 flex flex-row lg:flex-col pr-2 mb-4">
+              <Link
+                href={`/dashboard/${pathname
+                  .split("/")
+                  .at(2)}/settings/general`}
```

Comment:
`/dashboard/` should already be in pathanme. If you want to get the `[name]` use `props.params.name`